views/aggregate-flow-metadata.json

```diff
@@ -128,21 +140,24 @@
                     }
                 }
             },
-            "install_command": "pip install prefect-airbyte",
-            "description": {
-                "summary": "A flow that triggers a sync of an Airbyte connection and waits for it to complete.",
-                "returns": "`AirbyteSyncResult`: Model containing metadata for the `AirbyteSync`.",
-                "examples": [
-                    "Define a flow that runs an Airbyte connection sync:\n```python\nfrom prefect import flow\nfrom prefect_airbyte.server import AirbyteServer\nfrom prefect_airbyte.connections import AirbyteConnection\nfrom prefect_airbyte.flows import run_connection_sync\n\nairbyte_server = AirbyteServer(\n    server_host=\"localhost\",\n    server_port=8000\n)\n\nconnection = AirbyteConnection(\n    airbyte_server=airbyte_server,\n    connection_id=\"<YOUR-AIRBYTE-CONNECTION-UUID>\"\n)\n\n@flow\ndef airbyte_sync_flow():\n    # do some things\n\n    airbyte_sync_result = run_connection_sync(\n        airbyte_connection=connection\n    )\n    print(airbyte_sync_result.records_synced)\n\n    # do some other things, like trigger DBT based on number of new raw records\n```"
-                ]
-            }
+            "path_containing_flow": "prefect_airbyte/flows.py",
+            "slug": "run_connection_sync"
         }
     },
     "prefect-aws": {},
     "prefect-azure": {},
     "prefect-bitbucket": {},
     "prefect-census": {
         "run-census-sync": {
+            "collection_repo_url": "https://github.com/PrefectHQ/prefect-census",
+            "description": {
+                "summary": "A flow for triggering a Census sync run and waiting for completion.",
+                "returns": "The result of the sync run."
+            },
+            "documentation_url": "https://prefecthq.github.io/prefect-census/flows/#prefect_census.flows.run_census_sync",
+            "entrypoint": "prefect_census/flows.py:run_census_sync",
+            "install_command": "pip install prefect-census",
+            "logo_url": "https://images.ctfassets.net/gm98wzqotmnx/3oznRx2UFkd2XyqNkEZpzB/4e0967a828aec5e2527cedadf8d24e8a/llmjpn8a0pgu8szjmnyi.webp?h=250",
```

Comment:
How is the logo URL dynamically retrieved?

Reply:
from the blocks metadata view, happy to discuss in other PR for the script